python/bigipconfigdriver.py

```diff
@@ -609,51 +845,60 @@ def _do_reset(self):
                     # Manually create custom profiles;CCCL doesn't yet do this
                     if 'customProfiles' in config['resources']:
                         for profile in config['resources']['customProfiles']:
-                            _create_client_ssl_profile(self._bigip, profile)
+                            _create_client_ssl_profile(
+                                self._cccls[0]._bigip_proxy.mgmt_root(),
+                                profile)
                             customProfiles = True
 
-                    cfg = create_config_kubernetes(self._bigip, config)
-                    if self._bigip.regenerate_config_f5(cfg):
+                    incomplete = 0
+                    for cccl in self._cccls:
+                        cfg = create_config_kubernetes(cccl, config)
+                        try:
+                            incomplete += cccl._apply_config(cfg)
+
+                            # Manually delete custom profiles (if needed)
+                            if customProfiles:
+                                _delete_client_ssl_profiles(
```

Comment:
All of our special custom configurators (like _delete_client_ssl_profiles, _create_client_ssl_profiles and so on) should have the same error behavior as cccl._apply_config(cfg) .  I think this looks like:

- For any serious errors (failure to connect with CCCL, broken protocol, violated schema), throw a CCCL exception
- For any problem actually implementing a config on BIG-IP, return an incomplete counter.

Then our try/catch/Incomplete/backoff logic is the same for everything